gradio/components/gallery.py

```diff
@@ -97,8 +97,8 @@ def __init__(
         render: bool = True,
         key: int | str | tuple[int | str, ...] | None = None,
         preserved_by_key: list[str] | str | None = "value",
-        columns: int | list[int] | tuple[int, ...] | None = 2,
-        rows: int | list[int] | None = None,
+        columns: int | None = 2,
+        rows: int | None = None,
```

Comment:
Considering this has not worked since the start of 5.0, and that this format if providing a list of ints corresponding to different breakpoints also feels very un-pythonic and different from any of other our components, I think it's better to drop this behavior and go with this simpler api.